inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/reorg_yolo.cpp

```diff
@@ -7,63 +7,74 @@
 #include "single_layer_tests/reorg_yolo.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
+using namespace ov::test::subgraph;
 
-const std::vector<ngraph::Shape> inShapes_caffe_yolov2 = {
+const std::vector<ov::test::InputShape> inShapes_caffe_yolov2 = ov::test::static_shapes_to_test_representation({
     {1, 64, 26, 26},
-};
+});
 
-const std::vector<ngraph::Shape> inShapes = {
+const std::vector<ov::test::InputShape> inShapes = ov::test::static_shapes_to_test_representation({
     {1, 4, 4, 4},
     {1, 8, 4, 4},
     {1, 9, 3, 3},
     {1, 24, 34, 62},
     {2, 8, 4, 4},
-};
+});
 
-const std::vector<size_t> strides = {
-    2, 3
-};
+const std::vector<ov::test::InputShape> inShapesDynamic = {{{ngraph::Dimension(1, 2),
```

Comment:
minor
you can use `ngraph::Dimension(1, 2)` -> `{1, 2}` and `ngraph::Dimension::dynamic()` -> `-1`

Reply:
changed, Thanks!